pygmt/src/blockm.py

```diff
@@ -132,13 +127,10 @@ def blockmean(table, outfile=None, **kwargs):
 
 
 @fmt_docstring
+@use_common(options=["V", "a", "f", "r"])
 @use_alias(
     I="spacing",
     R="region",
-    V="verbose",
-    a="aspatial",
-    f="coltypes",
-    r="registration",
```

Comment:
Another question I have is the placement of common options like region (R) and frame (B). The current implementation seems to place them all at the very end after the non-common options. Is the `@use_common` decorator meant to be used only for parameters like `verbose`/`aspatial`/`panel`/etc placed at the end? Or do we want to use them for other common options like `region`/`frame` too?

I guess what I'm trying to ask is when we should use `@use_alias` vs `@use_common` :slightly_smiling_face:

Reply:
It could be an option to write a decorator that sorts the parameters section of the docstrings alphabetically (I would enjoy that, but maybe I am getting carried away here). Otherwise, I guess any common options that should go at the top or need unique descriptions for the given module (e.g., `region` in blockm*) could use `use_alias` while less important ones could use `use_common`.